Aptoide/bds-sdk
# Appc 1229 remove handshake from the poa star

**What does this PR do?**

This PR removes the hand shake of POA and now activates directly the WalletPoaService.
This PR also removes the bug of the POA duration be only 1 min and 50 secs and now its the real duration of 2 minutes.
Removed Tray dependency

**Database changed?**

No

**Where should the reviewer start?**

-PoAManager.java
-PoAServiceConnectorImpl.java

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APPC-1229](https://aptoide.atlassian.net/browse/APPC-1229)

This should be tested with the Appcoins dev wallet and with the toolbox in Debug mode.
Also the No wallet install flow and PoA starting after install wallet should be tested.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1229](https://aptoide.atlassian.net/browse/APPC-1229)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 
No



**Code Review Checklist**
